src/composables/sdk.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  IDefaultComposableOptions,
+} from '../types';
+import { useAccounts } from './accounts';
+import { IProtocol } from '../lib/protocols/IProtocol';
+import { AeternityProtocol } from '../lib/protocols/AeternityProtocol';
+import { BitcoinProtocol } from '../lib/protocols/BitcoinProtocol';
+import { DogecoinProtocol } from '../lib/protocols/DogecoinProtocol';
+
+let sdkRegistry: Record<string, IProtocol>;
+
+export function useSdk({ store }: IDefaultComposableOptions) {
+  const { activeAccount } = useAccounts({ store });
+
+  sdkRegistry = {
```

Comment:
This needs to be typed from the start.

Reply:
Already typed https://github.com/aeternity/superhero-wallet/blob/65e74e6f22e6b843e1001d924f8a7be0693013eb/src/composables/sdk.ts#L17